dashboard/views.py

```diff
@@ -460,7 +460,10 @@ def grade_distribution(request, course_id=0):
             'course_id': course_id,
             'enrollment_type': 'StudentEnrollment'
         })
-    if df.empty or df.count().current_grade < 6:
+    if len(df) <= config.GRADE_DISTRIBUTION_MINIMUM:
+        logger.error(f"Course enrollment count {len(df)} is lower than the minimum required to display the Grade Distribution visualization {config.GRADE_DISTRIBUTION_MINIMUM}")
```

Comment:
Can define a message string to be used for both line 464 and 465.

Reply:
OK, I have changed as requested 78eb5635a161fbca8e9143fb2c391879afb7d485